### PR TITLE
Add practice exercise Rail Fence Cipher

### PR DESCRIPTION
**Pull request by klimkin**

### CODE REVIEW

**Comment by vaeng** [comment]
@klimkin 
Hey, thanks for your effort. Can you correct the errors that were raised by the checks?

**Comment by vaeng** [comment]
@klimkin 
Do you want help with the results of the checks, or can you manage by yourself?

**Comment by klimkin** [comment]
> @klimkin Do you want help with the results of the checks, or can you manage by yourself?

If it still fails after the latest commit, I could use some help. Thank you!

File: exercises/practice/rail-fence-cipher/.meta/example.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#include "example.h"
```
Comment (vaeng) [inline exercises/practice/rail-fence-cipher/.meta/example.cpp:1]: This line must be changed to:
```suggestion
#include "rail_fence_cipher.h"
```
During the run of the test runner, files are renamed automatically.

File: exercises/practice/rail-fence-cipher/rail_fence_cipher_test.cpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#include "rail_fence_cipher.h"
+#ifdef EXERCISM_TEST_SUITE
+#include <catch2/catch.hpp>
+#else
+#include "test/catch.hpp"
+#endif
+
+TEST_CASE("encode with one rail, same output as input", "") {
```
Comment (vaeng) [inline exercises/practice/rail-fence-cipher/rail_fence_cipher_test.cpp:8]: Would you be so kind as to include the uuids of the tests? You can look at the other recently changed exercises for examples. Or ask if you have any questions on how to find them.

Comment (klimkin) [reply exercises/practice/rail-fence-cipher/rail_fence_cipher_test.cpp]: Done

**Review by vaeng — APPROVED** [review]
All checks pass. Congrats on your submission!

**Pull request merged**